src/ray/core_worker/transport/concurrency_group_manager.cc

```diff
@@ -95,7 +94,6 @@ void ConcurrencyGroupManager<ExecutorType>::Stop() {
   }
 }
 
-template class ConcurrencyGroupManager<FiberState>;
```

Comment:
Am I missing anything? Shouldn't we only have 1 fiber state (1 thread) per concurrency group. What do you mean by `1 thread per submitter worker`?

Reply:
Oh, we have actor scheduling queue per caller worker.